_pages/emergency-recovery/index.md

```diff
@@ -24,7 +24,113 @@ As there are "gotchas" in both of the below setups, we recommend you read [the a
+Step 1) Download and install Trezor Bridge. https://wiki.trezor.io/Trezor_Bridge  This has privacy concerns, see known issues. Alternatively, you can use this method.
+
+
+
+Step 2)
+Navigate to https://trezor.io/start/
+
+
+
+
+
+In this example you will select the trezor ONE.  You will restore the following seed you generated here.
+zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo buddy
+
+You previously entered this wallet into our Specter Desktop using a USB or DVD:
+
+{
+    "xfp": "669dce62",
+    "p2wsh": "Zpub74sb5KB3Ak1RwabGr8SHQnMTkd2mC3boVDgPf1jBFNxcXh7Nx4KV3XakPDtWLN5RpszdM7qcBN4wm7xreh8Ys2xYUBqQ9GtkTN8h5kRVecc",
+    "p2wsh_deriv": "m/48'/0'/0'/2'"
+}
+
+If you recall, you entered this wallet as “other”. To use the Trezor, you will change it to Trezor. You may also need to restart Specter Desktop to get this to work.  This way you can use the Trezor device to sign our transaction.
+
+
+
+
+Note: When you recover the wallet on the Trezor website, you will see the BIP 49 derivation path, but don’t worry, Specter Desktop will add the correct derivation.
+
+Step 3) Plug your Trezor into your computer.
+
+4. Click your device. It will tell you to inspect the package for tampering. Do all of that and click “continue to wallet.” If you are using a new device, skip to step 5. If you have used this wallet before, it will tell you to enter your pin. If it does, close that window because you are going to recover your seedpicker.net seed. You also want to make sure you send any funds you control with this wallet to another address before proceeding.
+
+Click wipe device
+Confirm you want to wipe your device. I am assuming there are no funds on this wallet.
+Confirm the wipe on your device.
+Wipe is completed, disconnect your device.
+Plug it back in:
+
+5. Click “Recover an existing wallet”. You are recovering 24 words, and you want to used advanced recovery because you NEVER TYPE OUR SEED WORDS INTO YOUB BROWSERS. You also did not use a passphrase with our seed picker wallet, so you will leave that unchecked. Click Continue. Agree to the terms of service in your device.
+
+7. To enter our seed into the device, you will navigate the scrambled BIP39 wordlist on our Trezor.  In this example I will enter “zoo” 23 times and “buddy” one time. The screen says, “Select letters for the 1st word” with 9 squares with dots bellow it. Each square with a dot in the middle represents one of the 9 digits on your keypad.  The top row represents 7, 8, and 9.  The middle row represents 4,5, and 6.  The bottom row represents 1, 2, and 3 on your keypad.
```

Comment:
You skipped `6.`, but actually it's easiest/best to make everything `1.` and jekyll will auto-increment.